rosa/cli.py

```diff
@@ -270,6 +273,28 @@ def build_execute_command(command, allowed_commands=None, aws_region=None):
     return execute_command(command=command)
 
 
+def aws_credentials():
```

Comment:
As for how it's implemented here, do you think we should move it to `cloud_tools` (for general use) and call it something like `set_aws_credentials `?

Reply:
Make sense
https://github.com/RedHatQE/cloud-tools/pull/80